tests/Magick.NET.Tests/Formats/Png/PngWriteDefinesTests/TheColorTypeProperty.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright Dirk Lemstra https://github.com/dlemstra/Magick.NET.
+// Licensed under the Apache License, Version 2.0.
+
+using ImageMagick;
+using ImageMagick.Formats;
+using Xunit;
+
+namespace Magick.NET.Tests;
+
+public partial class PngWriteDefinesTests
+{
+    public class TheColorTypeProperty
+    {
+        [Fact]
+        public void ShouldSetTheDefine()
+        {
+            using var image = new MagickImage();
+            image.Settings.SetDefines(new PngWriteDefines
+            {
+                ColorType = ColorType.Grayscale,
+            });
+            Assert.Equal("0", image.Settings.GetDefine(MagickFormat.Png, "color-type"));
+        }
+
+        [Fact]
+        public void ShouldNotSetTheDefineWhenNull()
+        {
+            using var image = new MagickImage();
+            image.Settings.SetDefines(new PngWriteDefines
+            {
+                ColorType = null,
+            });
+
+            Assert.Null(image.Settings.GetDefine(MagickFormat.Png, "color-type"));
+        }
+    }
```

Comment:
I think we should also add a test that checks that an exception is raised when an invalid `ColorType` is used.